index.md

```diff
@@ -72,6 +72,17 @@ can either be static or implement synaptic plasticity rules such as
 * [Synapses in NEST: An overview](synapses_overview)
 * [Developing synapse models](synapse_models)
 
+## Updating models from 2.14 or prior to 2.16
+
+With the introduction of the new connection infrastructure of the [5g kernel](https://www.frontiersin.org/articles/10.3389/fninf.2018.00002/full), 
+neuron, synapse and device models need to be slightly adapted from prior 
+versions to be compatible with the latest release (2.16). In the following we describe 
+all necessary changes:
+
+* [Updating models from 2.14 or prior to 2.16](model_conversion_5g.md)
```

Comment:
I like that the details are in a separate file, but I am not sure about the format of the link here: (i) Does a single-item bullet list make sense? (ii) Could the reader be confused because the link text is the same as the section heading? How about replacing the text from "In the following ..." with something like `[Please see these instructions for adapting models to NEST 2.16](model_conversion_5g.md)`?
@jessica-mitchell What do you think?

Reply:
I think the format matches quite nicely with the rest of the page (i.e. links are bullet lists), which is good for consistency and making the link stand out. However, I agree that the link text and title might be better if different from each other. What do you think of something like removing the headings "Updating models from  ..."  and putting both texts and links under a heading "Updating Models for the 5g Kernel"  or "Updating Models for Newest Release"?
Note: I am getting a 404 on the page for "updating models from 2.4 or prior to 2.6".

Reply:
I followed @jessica-mitchell  suggestion and changed it accordingly. Jessica, as you may have noticed, has all internal links point to 404, you would have to explicitly point to the file with the extension '.md', which is a problem for the HTML parser,